src/Raven.Client/Documents/Session/DocumentSession.IncludeCounters.cs

```diff
@@ -0,0 +1,31 @@
+//-----------------------------------------------------------------------
+// <copyright file="DocumentSession.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Raven.Client.Documents.Session.Loaders;
+
+namespace Raven.Client.Documents.Session
+{
+    /// <summary>
+    /// Implements Unit of Work for accessing the RavenDB server
+    /// </summary>
+    public partial class DocumentSession
+    {
+        public ILoaderWithInclude<T> IncludeCounter<T>(string name)
+        {
+            return new MultiLoaderWithInclude<T>(this).IncludeCounter(name);
+        }
+
+        public ILoaderWithInclude<T> IncludeCounters<T>(string[] names)
+        {
+            return new MultiLoaderWithInclude<T>(this).IncludeCounters(names);
+        }
+
+        public ILoaderWithInclude<T> IncludeCounters<T>()
+        {
+            return new MultiLoaderWithInclude<T>(this).IncludeCounters(new string[0]);
```

Comment:
Let's create an explicit value here, instead of using empty array.
Note that we might call `IncludeCounters` and then another call with specific value.

Reply:
maybe `IncludeAllCounters`?

Reply:
fixed